reinforcement_learning/rl_game_server_autopilot/sagemaker/common/docker_utils.py

```diff
@@ -0,0 +1,215 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
You don't have to include common folder files. They are already included in this repo